docs/application/dotnet/guides/user-interface/nui/vectorgraphics/Overview.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+VectorGraphics APIs are features for vector primitives drawing provided by NUI.
+<br>
+VectorGraphics namesapce provides [`CanvasView`](./CanvasView.md) component and `Shape`, `DrawableGroup`, and `Pricture` classes that inherit `Drawable` class.
+<br><br>
+The following lists the available feature using VectorGraphics:
+<br><br>
+## VectorGraphics Classes and Features
+
+- [CanvasView](./CanvasView.md)
+  - A component that use canvas area represented by viewbox.
+
+- [Shape & Stroke](./ShapeAndStroke.md)
+  - The set of path commands and shape informations. This can set color, gradient, and stroke information.
```

Comment:
If we are following British English, the correct spelling should be "colour". Also, the sentence is difficult to understand, perhaps it would be better to rewrite it.
```suggestion
  - The set of path commands and shape information. This can set color, gradient, and stroke information.
```